### PR TITLE
Definitions

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Checked with @SantiagoTorres looks good so far

### CODE REVIEW

File: supply-chain-security/compromises/compromise-definitions.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Types of Supply Chain Compromise
+
+This document serves to provide a consistent set of definitions, recognized by SIG-Security, for the Supply Chain Compromise Catalog.  The types of compromise may be expanded in the future as new or more distinct compromises occur.  When this happens, these definitions will be updated.
```
Comment (lumjjb) [inline supply-chain-security/compromises/compromise-definitions.md]: "recognized by SIG-Security" may be redundant here. I think anything in the repo should be governed by terms of use.

File: supply-chain-security/compromises/compromise-definitions.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+**Source Code Compromise** - occurs when source code repositories (public or private) is manipulated intentionally by the developer or through a developer or repository credential compromise.  Source Code compromised can also occur when source code in open-source dependencies is manipulated in a manner to make them more useful.
```
Comment (lumjjb) [inline supply-chain-security/compromises/compromise-definitions.md]: >manipulated in a manner to make them more useful.

Hmm, what dose it mean by more useful here?

File: supply-chain-security/compromises/compromise-definitions.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+_Reference(s)_:
+- [Mitre: Supply Chain Compromise](https://attack.mitre.org/techniques/T1195/)
+
+
+**Publishing Infrastructure** - occurs when the shipment, publishing, or distribution mechanism and infrastructure of the software is manipulated, exploited, or made unavailable.  This can result from a number of attacks that permit access to the infrastructure.
```
Comment (lumjjb) [inline supply-chain-security/compromises/compromise-definitions.md]: suggestion: when the integrity or the availability of X mechanisms are affected.

File: supply-chain-security/compromises/compromise-definitions.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- _Defeat_ - This kind of compromise can be deterred or defeated by implementing code-signing.  Code-signing requires attackers to perform multiple operations to be successful, making the level of effort higher.
```
Comment (lumjjb) [inline supply-chain-security/compromises/compromise-definitions.md]: suggestions: how about Protections/mitigations? Defeat seems like a very strong word that indicates that these are 100% coverage solutions.

File: supply-chain-security/compromises/compromise-definitions.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Types of Supply Chain Compromise
+
```
Comment (lumjjb) [inline supply-chain-security/compromises/compromise-definitions.md]: I think a list with anchor links to the different definitions in the doc would be nice

File: supply-chain-security/compromises/compromise-definitions.md (hunk 2 above)
Comment (lumjjb) [reply supply-chain-security/compromises/compromise-definitions.md]: nit: for consistency, the other definitions should follow "X compromise"

File: supply-chain-security/compromises/compromise-definitions.md (hunk 1 above)
Comment (TheFoxAtWork) [reply supply-chain-security/compromises/compromise-definitions.md]: corrected

File: supply-chain-security/compromises/compromise-definitions.md (hunk 2 above)
Comment (TheFoxAtWork) [reply supply-chain-security/compromises/compromise-definitions.md]: updated to include an explanation of more useful.  Not everything is technically a compromise in the definitions.  For example - Negligence is not a compromise.  One could argue that given the lexicons in use elsewhere, "compromise" should be dropped from the title of this definition and be left at "source code".  Thoughts?

File: supply-chain-security/compromises/compromise-definitions.md (hunk 3 above)
Comment (TheFoxAtWork) [reply supply-chain-security/compromises/compromise-definitions.md]: adjusted

File: supply-chain-security/compromises/compromise-definitions.md (hunk 5 above)
Comment (TheFoxAtWork) [reply supply-chain-security/compromises/compromise-definitions.md]: i suck at markdown but okay.

**Comment by TheFoxAtWork** [comment]
@lumjjb ready for ur review....

File: supply-chain-security/compromises/compromise-definitions.md (hunk 2 above)
Comment (lumjjb) [reply supply-chain-security/compromises/compromise-definitions.md]: > One could argue that given the lexicons in use elsewhere, "compromise" should be dropped from the title of this definition and be left at "source code". Thoughts?

Yea - i agree with this!

File: supply-chain-security/compromises/compromise-definitions.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+* [Dev Tooling](#dev-tooling)
+* [Negligence](#negligence)
+* [Publishing Infrastructure](#publishing-infrastructure)
+* [Source Code Compromise](#source-code-compromise)
+* [Trust and Signing](#trust-and-signing)
+
+## Source Code Compromise
+Occurs when a source code repository (public or private) is manipulated intentionally by the developer or through a developer or repository credential compromise.  Source Code compromised can also occur when source code in open-source dependencies is manipulated in a manner to make them more useful (more appealing for use in order to guarantee download of malicious content, trojan horse like).
```
Comment (lumjjb) [inline supply-chain-security/compromises/compromise-definitions.md]: Ah okay - i understand what you mean by this now. Would "Source code compromise can also occur with intentional introduction of security backdoors and bugs in Open Source code contributions by malicious actors"

**Comment by SantiagoTorres** [comment]
I like this, although I'm also not fully convinced if the word `defeat` is the term we'd like. In one hand I like that it does carries some strength/determination, but on the other I think it's not entirely clear what it means at first glance...

Other than that, pretty solid work @TheFoxAtWork !

File: supply-chain-security/compromises/compromise-definitions.md (hunk 2 above)
Comment (TheFoxAtWork) [reply supply-chain-security/compromises/compromise-definitions.md]: fixed

File: supply-chain-security/compromises/compromise-definitions.md (hunk 4 above)
Comment (TheFoxAtWork) [reply supply-chain-security/compromises/compromise-definitions.md]: resolved

File: supply-chain-security/compromises/compromise-definitions.md (hunk 6 above)
Comment (TheFoxAtWork) [reply supply-chain-security/compromises/compromise-definitions.md]: changed.

**Comment by SantiagoTorres** [comment]
I can't approve, but LGTM as well :)

**Comment by lumjjb** [comment]
@ultrasaurus @pragashj @dshaw This looks good to merge.. any additional comments?

**Comment by lumjjb** [comment]
Just waiting for additional comments from co-chairs. can discuss this at next meeting and merge this probably

**Comment by ultrasaurus** [comment]
This is fabulous.  Just one formatting issue before merging -- pls make the sections in the doc be the same order as the index.  Looks like that is alphabetical, which seems like a fine approach to me.

**Comment by TheFoxAtWork** [comment]
@ultrasaurus made the changes u requested.  @SantiagoTorres also added in the link to the definitions on the readme under compromises

**Comment by ultrasaurus** [comment]
Looks great -- thanks @TheFoxAtWork !

**Pull request merged**